demo/chicago-bikeshare-dashboard/run.py

```diff
@@ -27,12 +27,18 @@ def get_count_ride_type():
     """,
         con=connection_string,
     )
-    fig_m, ax = plt.subplots()
-    ax.bar(x=df["rideable_type"], height=df["n"])
-    ax.set_title("Number of rides by bycycle type")
-    ax.set_ylabel("Number of Rides")
-    ax.set_xlabel("Bicycle Type")
-    return fig_m
+    return gr.BarPlot.update(
```

Comment:
What it looks like now:

![image](https://user-images.githubusercontent.com/41651716/217665032-7cf6093b-d98f-4fe3-82dd-ba1516a2f3af.png)

Reply:
Beautiful!